resources/nl-core/nl-core-SOAPReport.SOAPLine.xml

```diff
@@ -15,7 +15,7 @@
     </telecom>
   </contact>
   <description value="A SOAP report is a textual report of a (partial) consult with relation to a single problem according to the SOAP structure. SOAP (acronym for subjective, objective, assessment, plan) is a method used by health professionals to structurally record information that comes up during contact between the patient and a health professionals in the patient's record. The following standardized format is used for reporting:&#xD;&#xA;&#xD;&#xA;* Subjective: the patient's complaint and request for help and the amnesic data&#xD;&#xA;* Objective: the findings from the physical and supplementary examination.&#xD;&#xA;* Assessment: the working hypothesis and the thinking process, for example a differential diagnosis of the healthcare professional.&#xD;&#xA;* Plan: the diagnostic plan or treatment plan and what has been discussed or agreed with the patient.&#xD;&#xA;&#xD;&#xA;The SOAP report is mainly used in general practice care.&#xD;&#xA;&#xD;&#xA;Note: the zib doesn't state explicitly if zib SOAPReport pertains a consult or a partial consult, but it has become clear that the zib, and thus this profile, should be interpreted as a partial consult with relation to a single problem (see &lt;https://bits.nictiz.nl/browse/ZIB-1474&gt;). The text above has been adjusted according to this insight." />
-  <purpose value="A derived profile from [zib-SOAPReport-Observation](http://nictiz.nl/fhir/StructureDefinition/zib-SOAPReport.SOAPLine) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <purpose value="A derived profile from [zib-SOAPReport-SOAPLine](http://nictiz.nl/fhir/StructureDefinition/zib-SOAPReport.SOAPLine) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />

```

Comment:
It should be zib-SOAPReport.SOAPLine instead of zib-SOAPReport-SOAPLine.

Reply:
Fix